flaml/nlp/hpo/hpo_searchspace.py

```diff
@@ -0,0 +1,321 @@
+from collections import OrderedDict
+
+from ..huggingface.trainer import TrainerForAutoTransformers
+from ray import tune
+from transformers import TrainingArguments
+
+from .grid_searchspace_auto import GRID_SEARCH_SPACE_MAPPING, AutoGridSearchSpace
+
+hp_type_mapping = {"learning_rate": [tune.sample.Float, "log"],
+                   "num_train_epochs": [tune.sample.Float, "linear"],
+                   "per_device_train_batch_size": tune.sample.Categorical,
+                   "weight_decay": [tune.sample.Float, "linear"],
+                   "warmup_ratio": [tune.sample.Float, "linear"],
+                   }
+
+def hpo_space_gridunion_continuous(logger,
+                                   model_type,
+                                   model_size_type,
+                                   dataset_name,
+                                   subdataset_name = None,
+                                   **custom_hpo_args):
+    gridunion_space = hpo_space_gridunion(logger, model_type, model_size_type, dataset_name, subdataset_name)
+    gridunion_space_continuous = {}
+    for each_hp in hp_type_mapping.keys():
```

Comment:
use 
for each_hp, value in hp_type.mapping.items():
and replace hp_type_mapping[each_hp] with value

Reply:
removed